src/types/message/index.js

```diff
@@ -41,7 +39,7 @@ class BitswapMessage {
   }
 
   addBlock (block) {
-    assert(Block.isBlock(block), 'must be a valid cid')
+    assert(Block.isBlock(block), 'must be a valid block')
```

Comment:
Can we remove `assert` here ? it's big for the browser and it doesn't add much value.

Reply:
Death to types.